src/CustomerBundle/Controller/CustomerController.php

```diff
@@ -0,0 +1,146 @@
+<?php
+
+namespace CustomerBundle\Controller;
+
+use Symfony\Bundle\FrameworkBundle\Controller\Controller;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route;
+
+
+use Symfony\Component\HttpFoundation\Request;
+use CustomerBundle\Entity\Customer;
+use Symfony\Component\Form\Extension\Core\Type\TextType;
+use Symfony\Component\Form\Extension\Core\Type\TextareaType;
+use Symfony\Component\Form\Extension\Core\Type\DateTimeType;
+use Symfony\Component\Form\Extension\Core\Type\ChoiceType;
+use Symfony\Component\Form\Extension\Core\Type\SubmitType;
+
+class CustomerController extends Controller
+{
+    /**
+     * @Route("/manage_customer", name="manage_customer")
+     */
+    public function indexAction(Request $request)
+    {   
+        $customers = $this->getDoctrine()
+                ->getRepository('CustomerBundle:Customer')
+                ->findAll();
+        
+        return $this->render('CustomerBundle:CustomerManage:index.html.twig', array(
+            'customers' => $customers
+        ));
+    }
+    
+    /**
+     * @Route("/manage_customer/create", name="manage_customer_create")
+     */
+     public function createAction(Request $request)
+    {  
+        $customer = new Customer;
+

```

Comment:
o mamo a co to na dole, fakkkkkkkkkkkkkk oddzielna klasa ! i zamist array() dajemy []